packages/core/src/index.ts

```diff
@@ -28,3 +28,5 @@ export { KeepAliveManager } from "./lib/keep_alive_manager.js";
 export { StreamManager } from "./lib/stream_manager.js";
 
 export { MetadataCodec, wakuMetadata } from "./lib/metadata/index.js";
+
+export * from "./lib/message_hash.js";
```

Comment:
Let's re-export it from `@waku/sdk`

Reply:
it is being exported from `@waku/sdk` under `waku` as 
```export * as waku from "@waku/core";```

do you prefer a more explicit export? let me know! happy to make changes

Reply:
I am not sure, let's keep it as it is for now, later we can lift export level of it.